### PR TITLE
chore(release): publish packages

### PR DESCRIPTION
**Pull request by provokateurin**
- dynamite@0.5.0+1
 - dynamite_runtime@0.5.0+1
 - nextcloud@8.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,66 @@
...
+ - **BREAKING** **FEAT**(nextcloud): Update to notifications 30. ([d9795186](https://github.com/nextcloud/neon/commit/d9795186be62b4c6715c5663f8b8809bfda091a7))
+ - **BREAKING** **FEAT**(nextcloud): Update to server 30. ([16c6272b](https://github.com/nextcloud/neon/commit/16c6272b23cfd0241fa5b22762e0850c5525cd60))
+ - **BREAKING** **CHORE**(nextcloud): Remove deprecated userAgent and cookieJar parameters from NextcloudClient. ([4ce78c1f](https://github.com/nextcloud/neon/commit/4ce78c1f68ac1fb550ba4e41f3cdc6297ccccdc7))
+ - **BREAKING** **CHORE**(nextcloud): Remove deprecated AppIDs. ([ab2fb5a2](https://github.com/nextcloud/neon/commit/ab2fb5a2a1f36b7e5c50facb8b5699ae09185315))
+
+#### `dynamite` - `v0.5.0+1`
+
+ - **REFACTOR**: change package file structure. ([6696e6ea](https://github.com/nextcloud/neon/commit/6696e6ead1751462e233afb5eab22a458e0c9e73))
+ - **FIX**(dynamite): Escape HTML in docs. ([063730f4](https://github.com/nextcloud/neon/commit/063730f448be7c9806183a844ad3fab4275e8580))
+ - **FIX**(dynamite): Escape wireNames. ([5915c570](https://github.com/nextcloud/neon/commit/5915c57035879db20ba7700924dffaf70255b549))
```
Comment (provokateurin) [inline CHANGELOG.md:59]: This looks wrong to me, the new version should be `0.5.1` since there were fixes

Comment (Leptopoda) [reply CHANGELOG.md:59]: Looks right to me.
With 0.5.0 being < 1.0.0 we are in the range where we'd use the `+`

Comment (provokateurin) [reply CHANGELOG.md:59]: True I overlooked that, but it should still increase the minor version.

**Review by Leptopoda — APPROVED** [review]
:tada:

File: CHANGELOG.md (hunk 1 above)
Comment (provokateurin) [reply CHANGELOG.md:59]: FFS I forgot the Dart convention just shifts everything down by one place. Then this is actually correct :woman_facepalming:

**Comment by provokateurin** [comment]
Tests fail with `SqfliteFfiException(error, Invalid argument(s): Failed to load dynamic library 'libsqlite3.so': libsqlite3.so: cannot open shared object file: No such file or directory}) DatabaseException(Invalid argument(s): Failed to load dynamic library 'libsqlite3.so': libsqlite3.so: cannot open shared object file: No such file or directory)` but https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md says sqlite3 is installed.

**Pull request merged**